### PR TITLE
ExperimentSetsTableTabViewTitle Edits

### PR DESCRIPTION
**Pull request by utku-ozturk**
`ExperimentSetsTableTabViewTitle` used to be a customized title for **Experiment Sets** embedded search tables but its scope is now expanding for `EmbeddedItemSearchTable`:

1. `ExperimentSetsTableTabViewTitle` is renamed as `SearchTableTitle`.
2. added into the placeholders that allows the jsx component to be embedded in static sections.
3. gets a `title` prop to customize search type text.

PS: `embedded-search-table-demo.jsx` is also updated.

### CODE REVIEW

File: src/encoded/static/data/embedded-search-table-demo.jsx [hunk 1]
```diff
@@ -120,5 +121,29 @@
...
+
+    <p>Set <code>{"title={null}"}</code> or do not define <code>title</code> at all.</p>
+
+    <h4 className="mt-2">Default Title</h4>
+
+    <p>
+        The built-in <code>SearchTableTitle</code> component renders the total count of type (you should also pass it as title prop into SearchTableTitle) and an optional Browse/Search button that redirects you to Browse or Search pages with respectively.
+    </p>
+
+    <pre className="border rounded px-3 py-2">{'<EmbeddedItemSearchTable\r\n        searchHref="\/search/?type=Biosource"\r\n        schemas={schemas}\r\n        session={session}\r\n        facets={null}\r\n        title={<SearchTableTitle title="Biosource" externalSearchLinkVisible={true} />}\r\n    />'}</pre>
```
Comment (j1z0) [inline src/encoded/static/data/embedded-search-table-demo.jsx:136]: Codacy found an issue: [Unnecessary escape character: \/.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6302309)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6302309&bid=20495672)

**Review by alexkb0009 — APPROVED** [review]
Looks great!

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6302309&bid=20495672)

File: src/encoded/static/data/embedded-search-table-demo.jsx (hunk 1 above)
Comment (j1z0) [inline src/encoded/static/data/embedded-search-table-demo.jsx:136]: Codacy found an issue: [Unnecessary escape character: \/.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6302309)

**Comment by j1z0** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6302309&bid=20495672)

File: src/encoded/static/data/embedded-search-table-demo.jsx (hunk 1 above)
Comment (j1z0) [inline src/encoded/static/data/embedded-search-table-demo.jsx:136]: Codacy found an issue: [Unnecessary escape character: \/.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=6302309)

**Pull request merged**